packages/server/integrations/helpers/authorizeOAuth2.ts

```diff
@@ -32,7 +32,12 @@ const transformBody = (contentType: string, body?: Record<string, string>): stri
 }
 
 export const authorizeOAuth2 = async <
-  TSuccess = {accessToken: string; refreshToken: string | undefined; scopes: string | undefined}
+  TSuccess = {
+    accessToken: string
+    refreshToken: string | undefined
+    scopes: string | undefined
+    expires_in: number | undefined
```

Comment:
-1 we're fixing the other fields names to camelCase, so please do it also for `expiresIn`